### ISSUE TITLE
Background images for views according to mockups

### ISSUE THREAD

**Issue by l-marchesi** [Kennox/rscc#58]
see here:
https://dzone.com/articles/styling-javafx-control-css

```
    -fx-background-image: url("images/background.png");
    -fx-background-repeat: no-repeat;
    -fx-background-position: left bottom;
```

**Comment by l-marchesi** [comment]
One view is done (RequestHelp) but the background for SupportView is no svg. We should try with the normal provided png from @pwigger

**Comment by martinfrancois** [comment]
fixed by PR #91
